debug_toolbar/panels/cache.py

```diff
@@ -5,11 +5,13 @@
 
 from django.conf import settings
 from django.core import cache
-from django.core.cache import CacheHandler, caches as original_caches
+from django.core.cache import CacheHandler
+from django.core.cache import caches as original_caches
 from django.core.cache.backends.base import BaseCache
 from django.dispatch import Signal
 from django.middleware import cache as middleware_cache
-from django.utils.translation import gettext_lazy as _, ngettext as __
+from django.utils.translation import gettext_lazy as _
+from django.utils.translation import ngettext as __
```

Comment:
I have never seen this convention until now (`__ = ngettext`). I don't necessarily disagree with it but I'm not sure it's well-known enough.

Reply:
I restored the `combine_as_import` option so this line remains the same as before.

Reply:
Yes, I agree. In this case however I think keeping `combine_as_imports` is preferable. 

The `django` profile also uses `combine_as_imports` and the resulting diff is smaller.